MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include .coveragerc
+include CHANGELOG.md
+include COPYING.txt
+include Makefile
+include README.md
+
+graft gneiss
+
```

Comment:
Ok.  Then let's not include them then.

On Tue, Aug 30, 2016 at 10:17 AM, Daniel McDonald <notifications@github.com>
wrote:

> In MANIFEST.in
> <https://github.com/biocore/gneiss/pull/44#discussion_r76838918>:
>
> > @@ -0,0 +1,11 @@
> > +include .coveragerc
> > +include CHANGELOG.md
> > +include COPYING.txt
> > +include Makefile
> > +include README.md
> > +
> > +graft gneiss
> > +
>
> i dont either, just checking if you wanted them
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/gneiss/pull/44/files/6870b42da40d27dfc6a713ae20d12ccd4ff4961a#r76838918>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD_a3SPMQf2YW0WgOCkT9Ulmr7_wyLJjks5qlGWOgaJpZM4JuYBq>
> .
>